tools/wptrunner/wptrunner/environment.py

```diff
@@ -94,6 +94,15 @@ class TestEnvironment:
     def __init__(self, test_paths, testharness_timeout_multipler,
                  pause_after_test, debug_test, debug_info, options, ssl_config, env_extras,
                  enable_webtransport=False, mojojs_path=None, inject_script=None):
+        # this specific version change is required as setting sys.stdout or
+        # sys.stderr will cause mozlog to not work properly. ".reconfigure"
+        # is only available after python 3.7, but since the builders are
+        # already all using 3.8+ we're just fixing for only 3.7+
+        # TODO(crbug.com/1378771) once we no longer support < 3.6 remove the
```

Comment:
For future reference, I would avoid Chromium-specific references like `crbug.com` or "builder". If this is a `TODO` for `wptrunner`, let's file it against web-platform-tests/wpt as a GitHub issue.

Reply:
yeah makes sense I'll add it against web-platform-tests/wpt as a github issue.

Reply:
So looking at I would think that the issue will be fixed, but we need to watch for an official mozilla release. Also since I can't really test mozilla changes directly on our builder, I think what I'll do is leave this change as is and for the TODO I'll add that we should remove this entire change if the issue is resolved after mozilla officially pushes a release.